### PR TITLE
[Vectorize] Complete Vectorize 2 limits

### PR DESCRIPTION
**Pull request by netgusto**
### Summary

Adding metadata index limits to Vectorize 2 limits page.

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/vectorize/platform/limits.mdx [hunk 1]
```diff
@@ -27,6 +27,8 @@ The following limits apply to accounts, indexes and vectors (as specified):
 | Maximum namespaces per index                                  | 1000 namespaces <sup>beta</sup>                                  |
 | Maximum namespace name length                                 | 64 bytes                                                         |
 | Maximum vectors upload size                                   | 100 MB                                                           |
+| Maximum metadata indexes                                      | 10                                                               |
```
Comment (garvit-gupta) [inline src/content/docs/vectorize/platform/limits.mdx]: nit: `Maximum metadata indexes per Vectorize index`

Comment (netgusto) [reply src/content/docs/vectorize/platform/limits.mdx]: good shout; fixed

**Review by garvit-gupta — APPROVED** [review]

File: src/content/docs/vectorize/platform/limits.mdx [hunk 2]
```diff
@@ -27,6 +27,8 @@ The following limits apply to accounts, indexes and vectors (as specified):
 | Maximum namespaces per index                                  | 1000 namespaces <sup>beta</sup>                                  |
 | Maximum namespace name length                                 | 64 bytes                                                         |
 | Maximum vectors upload size                                   | 100 MB                                                           |
+| Maximum metadata indexes per Vectorize index                  | 10                                                               |
+| Maximum indexed data per metadata index per vector            | 64 bytes                                                         |
```
Comment (elithrar) [inline src/content/docs/vectorize/platform/limits.mdx:31]: Do we need to flesh this out more in the metadata docs?

- https://developers.cloudflare.com/vectorize/get-started/intro/#4-optional-create-metadata-indexes
- https://developers.cloudflare.com/vectorize/reference/metadata-filtering/

Comment (netgusto) [reply src/content/docs/vectorize/platform/limits.mdx:31]: Definitely, I'll update the PR.

**Review by sejoker — APPROVED** [review]

File: src/content/docs/vectorize/platform/limits.mdx (hunk 2 above)
Comment (netgusto) [reply src/content/docs/vectorize/platform/limits.mdx:31]: Done.

**Review by elithrar — APPROVED** [review]

**Pull request merged**